lib/JMX/Jmx4Perl/Agent.pm

```diff
@@ -344,6 +344,8 @@ sub request_url {
         $req .= "/" . $self->_escape($self->_null_escape($request->get("value")));
         $req .= $self->_extract_path($request->get("path"));
     } elsif ($type eq LIST) {
+        # The (URI escaped) colon after the <domain> must be transformed into a slash
+        $req =~ s|%3A|/|;
```

Comment:
Could we make the replacement case insensitive please ?

Reply:
Sure. I did not do it because URI::Escape (used by `_escape`) uses uppercase but it is probably safer this way.